src/app/games/checkers/common/CheckersFailure.ts

```diff
@@ -0,0 +1,33 @@
+import { Localized } from 'src/app/utils/LocaleUtils';
+
+export class CheckersFailure {
+
+    public static readonly CANNOT_GO_BACKWARD: Localized = () => $localize`You cannot go backward with normal pieces!`;
+
+    public static readonly CANNOT_SKIP_CAPTURE: Localized = () => $localize`You must capture when it is possible!`;
+
+    public static readonly MUST_FINISH_CAPTURING: Localized = () => $localize`You must finish this capture!`;
+
+    public static readonly THIS_PIECE_CANNOT_MOVE: Localized = () => $localize`This piece cannot move!`;
+
+    public static readonly FRISIAN_CAPTURE_MUST_BE_EVEN: Localized = () => $localize`Frisian capture must be of even length!`;
+
+    public static readonly INVALID_FRISIAN_MOVE: Localized = () => $localize`This is an invalid orthogonal move, frisian capture must be at least steps of 4. Look at the indicators to help you!`;
```

Comment:
a frisian capture*
"must be at least steps of 4" c'pas très clair, en particulier sur un message qui pop vite sur l'écran non ?

Reply:
Que propose-tu ? 
(à part de prendre le ticket "durée d'affichage proportionnelle au nombre de mots")
C'est vrai qu'à mon avis, un BOOON pourcentage des messages toasts sont jetés à la poubelle non lu, gaspillage !